src/components/SideBar.js

```diff
@@ -200,14 +200,16 @@ const NotificationMenu = memo(
           <Badge>{countUnseenAnnouncements}</Badge>
         ),
         onClick: onClickAnnouncementsButton,
+        visible: i18n.language === 'nl',
```

Comment:
@geoffreydhuyvetters @simon-debruijn is this okay with the `.filter()` or should we add some kind of `visible` prop on box and use that instead? I couldn't find one in Storybook so I assume it doesn't exist yet.

Reply:
Ah, I think I can just add a prop to `MenuItem`

Reply:
See https://github.com/cultuurnet/udb3-frontend/pull/176/commits/e977e6c5482d19d3cd424ff0b3277be6ade4b84b, but I can still revert that if the first solution was better.

Reply:
I prefer the last solution

Reply:
Do you think it would be valuable to add the `visible` prop to `Box` while we're at it? cc @geoffreydhuyvetters ?

Reply:
I was planning on doing that idd, makes sense to have it everywhere IMO.

Reply:
We can merge this one as is